oiosaml/src/main/java/dk/gov/oio/saml/servlet/DispatcherServlet.java

```diff
@@ -214,7 +215,11 @@ protected void doPost(HttpServletRequest req, HttpServletResponse res) throws Se
         }
 
         try {
-            samlHandler.handlePost(req, res);
+            if (StringUtil.isNotEmpty(req.getHeader("SOAPAction"))) {
+                samlHandler.handleSOAP(req, res);
+            } else {
+                samlHandler.handlePost(req, res);
+            }
 		}
 		catch (ExternalException | InternalException e) {
```

Comment:
There are some reformatting must occur here.

Reply:
As soon as we have logging in master I will reformat everything on a separate branch and make a pull-request with this. It will create too much confusion and merge conflicts to fix this with other changes...